zulipterminal/ui.py

```diff
@@ -297,6 +291,20 @@ def keypress(self, size: urwid_Box, key: str) -> Optional[str]:
             key = "end"
         return super().keypress(size, key)
 
+    def render(self, size: urwid_Box, focus: bool) -> Any:
+        if not self.controller.autohide:
+            maxcols, maxrows = size
+            if maxcols < 102 and not self.controller.pseudo_autohide:
+                self.controller.pseudo_autohide = True
+                self.show_right_panel(visible=False)
+                self.body.focus_position = 0
+            elif maxcols > 102 and self.controller.pseudo_autohide:
+                self.show_right_panel(visible=True)
+                self.show_left_panel(visible=True)
+                self.controller.pseudo_autohide = False
+
```

Comment:
What would happen in the `else` statement that you are suggesting? The second part of the conditional is basically so that every render call should not change body.contents. Most of the time the render call is going to enter `else`. (Many time actually)

Reply:
Nothing happens at 102 but there may be different behavior while increasing or decreasing width. It better to make it fall into the autohide category

Reply:
Not sure but can we define a constant variable for 102? The variable name could justify many things

Reply:
Sure, I'll add them.